codeHF/workflows.yml

```diff
@@ -83,6 +83,17 @@ workflows:
       default: [HFCANDBPLUSBASE, HFCANDBPLUSEXT]
       mc: [HFCANDBPMCREC, HFCANDBPMCGEN]
 
+  o2-analysis-hf-candidate-creator-lb:
+    <<: *cand_creator
+    options:
+      mc: "--doMC"
```

Comment:
This should be removed together with replacing the `--doMC` option by a `processMC` switch ASAP.